staging/src/k8s.io/apimachinery/pkg/util/remotecommand/constants.go

```diff
@@ -46,8 +46,23 @@ const (
 	// adds support for exit codes.
 	StreamProtocolV4Name = "v4.channel.k8s.io"
 
+	// The subprotocol "v5.channel.k8s.io" is used for remote command
+	// attachment/execution. It is the 5th version of the subprotocol and
+	// adds support half-closing STDIN and a heartbeat.
+	StreamProtocolV5Name = "v5.channel.k8s.io"
```

Comment:
long term we may think in make it official https://www.iana.org/assignments/websocket/websocket.xml#subprotocol-name, the protocol is not very complicated